flake.nix

```diff
@@ -3,13 +3,15 @@
 
   inputs = {
     flake-parts.url = "github:hercules-ci/flake-parts";
-    nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
+    nixpkgs.url = "github:NixOS/nixpkgs/master";
 
-    # override for boost -> std shared_ptr
-    crocoddyl = {
-      url = "github:loco-3d/crocoddyl/release/3.0.0";
-      inputs.flake-parts.follows = "flake-parts";
-      inputs.nixpkgs.follows = "nixpkgs";
+    mim-solvers = {
```

Comment:
add a comment to document when this can be removed : after the next mim-solvers release reach nixpkgs/nixos-unstable